waiter/config-full.edn

```diff
@@ -752,6 +752,16 @@
  ; ---------- Load Balancing ----------
 
  :work-stealing {
+                 ;; Work-stealing maintains a simple moving average of successful offers which is
+                 ;; used as feedback to determine the probability of making subsequent offers.
+                 :moving-average {;; the initial value of the moving average
+                                  :initial-value 1.00
+                                  ;; the maximum value of the moving average
+                                  :maximum-value 1.00
```

Comment:
Can we use `max` and `min`, in line with how we define all other maximums and minimums in our config?

Reply:
Will rename to `max` and `min`.